cpp/dolfinx/fem/DirichletBC.cpp

```diff
@@ -451,7 +455,11 @@ std::array<std::vector<std::int32_t>, 2> fem::locate_dofs_topological(
 //-----------------------------------------------------------------------------
 std::vector<std::int32_t> fem::locate_dofs_geometrical(
     const FunctionSpace& V,
-    const std::function<xt::xtensor<bool, 1>(const xt::xtensor<double, 2>&)>&
+    const std::function<std::vector<std::int8_t>(
+        std::experimental::mdspan<
```

Comment:
Is this the same as `cmdspan3x_t` ?

Reply:
The problem is that older version of doxygen, i.e. what we have in the test image, seem to match strings rather than the underlying type when checking that the declaration in the header match what is in the implementation file.

Reply:
doxygen checks fails when using the typedef.